toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1283,6 +1295,10 @@ PreferencesPopup::PreferencesPopup()
       new CheckBox(tr("Minimize Savebox after Editing"), this);
   m_useNumpadForSwitchingStyles =
       new CheckBox(tr("Use Numpad and Tab keys for Switching Styles"), this);
+  m_keepFillOnVectorSimplifyCB = new CheckBox(
+      tr("Keep fill when using \"Replace Vectors\" command"), this);
+  m_useHigherDpiOnVectorSimplifyCB = new CheckBox(
+      tr("Use higher DPI for calculations - Slower but more accurate"), this);
```

Comment:
@turtletooth 
Sorry for the late notice, but how about opening some dialog each time you trigger the `Replace Vectors with Simplified Vectors` command, containing these options instead of put them in the preferences popup?
Since the preferences popup has already many items and the `Replace Vectors with Simplified Vectors` command seems not to be used so quickly as it cannot have the dialog before execution.

Reply:
> However, I actually use the commands often (there are still often fill issues) and for workflow, being able to minimize clicks is nice.

OK, I didn't imagine that. Then putting these options in the preferences is reasonable.